sdk/src/dlob/DLOB.ts

```diff
@@ -733,12 +815,12 @@ export class DLOB {
 
 		// All bids/asks that can expire
 		const bidGenerators = [
-			nodeLists.limit.bid.getGenerator(),
+			nodeLists.restingLimit.bid.getGenerator(),
```

Comment:
i feel like taking limit orders can expire too...if they set the order TTL similar to auction duration? or were you planning expire orders the next time updateRestingLimitOrders happens?